cerberus-cli/src/main/java/ca/on/oicr/gsi/cerberus/Cerberus.java

```diff
@@ -61,35 +71,79 @@ public static void main(String[] args) {
             System.exit(1);
         }
 
-        if (line.hasOption("help")) {
-            new HelpFormatter().printHelp("java Cerberus", options);
+        if (line.hasOption(HELP_OPT)) {
+            new HelpFormatter().printHelp("java -jar cerberus-cli/target/cerberus-cli-${VERSION}-jar-with-dependencies.jar ", options);
             System.exit(0);
         }
 
+        Boolean error = false;
+        // check required options are present
+        ArrayList<String> required = new ArrayList<>();
```

Comment:
By Java convention, it's preferred to use the `List` and `Map` interfaces even when filling it with a particular subclass.